parity/rpc.rs

```diff
@@ -81,6 +82,7 @@ impl Default for IpcConfiguration {
 				let data_dir = ::dir::default_data_path();
 				parity_ipc_path(&data_dir, "$BASE/jsonrpc.ipc", 0)
 			},
+			chmod: 660,
```

Comment:
Can it be this?
```suggestion
			chmod: 0o660,
```

Reply:
actually, no.
It'll be converted to octal here:  https://github.com/paritytech/parity-ethereum/pull/11273/files#diff-213a05445a46c1e77ad474cd0098bd18R241-R242